namespaces/live.cloud-platform.service.justice.gov.uk/laa-information-exchange-development/04-networkpolicy.yaml

```diff
@@ -3,6 +3,8 @@ kind: NetworkPolicy
 metadata:
   name: default
   namespace: laa-information-exchange-development
+  annotations:
```

Comment:
These annotations belong to the ingress definition and not on networkpolicy

Reply:
Closing the PR as this should all be covered in the helm deploy files.